app/password/reset/route.js

```diff
@@ -2,18 +2,26 @@ import Ember from 'ember';
 import DisallowAuthenticated from "../../mixins/routes/disallow-authenticated";
 
 export default Ember.Route.extend(DisallowAuthenticated, {
-  model: function(){
+  model() {
     return this.store.createRecord('password-reset-request');
   },
 
+  resetController(controller, isExiting) {
```

Comment:
In fact I don't think you need to check the `isExiting` boolean, right? b/c you would want these things to be reset even if there was a transition to this same route.